doc/non-functional tests/performance.md

```diff
@@ -0,0 +1,26 @@
+# Teljesítmény mérés
+
+JVM alapú programok memória és CPU idő használatát lehet mérni profiling programok segítségével, például a [VisualVM](https://visualvm.github.io/). 
+
+Az alábbi mérések egy ~5 perces futás eredménye.
+
+## CPU idő
+
+![cpu time](images/cpu.png)
+
+Mint látható a méréseken a cpu idő esemény vezérelt programoknál elhanyagolható mértékű, mivel a program legtöbb ideje várakozással telik.
```

Comment:
"Mint látható a méréseken a cpu idő esemény vezérelt programoknál" -> "Mint látható a méréseken, a CPU-idő eseményvezérelt programoknál"